test/middleware/authentication.middleware.spec.ts

```diff
@@ -55,7 +54,7 @@ describe('Authentication controller', () => {
     expect(res.redirect).toHaveBeenCalledWith(signinRedirectPath);
 
     expect(logger.infoRequest).toHaveBeenCalledTimes(1);
```

Comment:
You could also test that the call to `logger.info` is made here (with that different status code output).

Reply:
Working on that, thanks :)